pkg/ampmail/accountPasswordConfirm.go

```diff
@@ -0,0 +1,19 @@
PV/EV6l6zw6glwd8koX7oURxMMgkKQBhsYr6r/Zr/AOCU3wA/Ye8OaGunaLpZ+JOpaabc61rdy2oXyFl3TiF2YeTGCGAK7cj5cnOD7FfOlUpczg+dJNK1k7Jab/ev+AePWzCqsY8RTflby0aT/M/NfxF8AdD8JXg0a5+FfiiObSgIJZtOsneCeTAJIYbF6FTgZwGHPIJK3f8Agp//AMFIPhL+x9+0H/wg9xYyahc2Vubme00rRP7Nm015GLbJkOD8ybGQN8wBYkLuAorwoZhm81zezX4HtTzqjOXPJJN9FeyPk/wp8GpPhj451iT4f6n4quNUtNdvoZNKihK6NpqMyosDlnZZjFGAquu4gfSrXiLwh4t+I2izalFb+I9Ps/D1oryQX+pXc2buZw1p86thmbDsAx2qCCWX5RXU/Erxfb+HdP023sxf6Xfa9BEj/wBhXulxWNqZSS7SqryOcsW/1XmcPw55xS+M/iHxF+zp4h1zQ/EHiS41jwnocWlw3kMCyxwX7TxI8DeeECKY02Aq6AZHBPQe3Sl7Sltr1a/LX/K55UHGFT2kmtNlrq/l+exzH7PPjn+wf2i9fs7yVY4tWlW5j2DYrvsG8hVJAGcjHH3T719Y3/xmtfAPgWSR9slwxIiHVmJ6DHpXwPbfE7Rdd+JSeItKaWV4yYPIjVFjVghACYOQMAZLAAk55FS/tMfG7XPENhpMem+e2m3m3yZ4ozuklU4kC9cbcDj3rgxGDVSveGlz9ByXPZYLLnCprJP8HqfZ3wD+I+mXvxS0u317XLWDUriX7T9j3/Nb53BT29Bz681+m3hf4h6RoUGhafa+LNHOva8irZ2Wd0l5KQdsYHIye5OBx1r+cbwv8LvG/jzxHa6lb6jBez2qytZl7wR3Ac9Ixux82QDyf5c/U3wV0344fD/RfD+oW95faXrkQMU11dXMbtFAC6qpVZNwYBixIycdOM0f2bTb+JGOdcQYytGMVQlZa7XuvPQ/aX4ffGGz1b4haloeuW8+j+ItMMkVxbSNtbK8FhjhvqMivCf29/2gbjwBrPguSwtjeXFjrEcskUUpT7RAofzVD4+UmPd6HPQggEfEnws/ax+Iuo/tK6P4X8RyapeatNOLvTbhbdgqwrjePMIG+NgSOM4HpXRf8FBP2vP+FWeFrvxNZ2MtxcaXqaQWSvKgjkmlR1PUNvRV3BwQOJOCCQa5/qU41VFarXY+dxGd060UmuR3Sd/xff5Ho37X/wDwUJ+Ff7KnwO8RePvB2reJ28aNawWiaHaa3cy6fsmZvLsporueTyoZVLystrFEMZZtxIz+Hf7R37RnxA/bP+LetfEL4neLNW17xdqhLJPPIwhtot7MLa3UHEECbm2RoAoyeMkk6kOppqOsy3WpBtTgvJlmuRK28hum4ZPUAfkMV1Hxl1Twq/hX7Ro9n5+pSweXHGTlCxOcqB06kDt3r1/ZLDtJ+8+/6Hz0cPTrKdWErWd7dfX7kfOfibxPfeKtZkvdVv77VLx1SNp7y4e4mKooVVLuSSFUAAZwAABRUfivS5dA8TX1nJJayyW8mxmh3bM+24A/mBRXfGKaueRLc/QD4wftVeOvFXh77Hp/h25urWzDCZobsw/u5V+dZYcgq46c5GO/SvCfHXxg8bWi/bfGNhdXVtqjwStDe3ipZ3cFuUCLsVxuUFMZBzknngiuV+J37XPizxtLK8RsdIEgKM9nFtndemDIST+WK8e1zU7vxFqjXF/cXV5MBl3nlaRj36sc1y4DC1qMOWenXuehisTSdXmpa/h+B7z4b+MPhXwf4YuNfvL7z9avpWl07RdNjPl6btbh5ZW5AHQICcg9a9I8EePG1i6t7eOJbnSIb4X1u5TcSCCSPl+pP1r4xuf9ZMNvVAR9OK9y/ZYsNY8V6Jqy2t40i6ZcQKltITtw248fTaBjoR+FbV6cbXOzKcZU9r7OOt9bemrPtjwX8T9L8CeIYZPFHheXUIboqI7qGNGEoz8u85BUjgEg88e9fWn7Nfx18OeNr9tD8H+AtR1TUsBy12dsMD9TlnLYA4yRg8ivjPwB4nt/F+nL4f1m5jKWZdIbaSPl4lw27KnKMB04/Gvpz4VfFC28B+BLWz8Jro+k6rNI1rcX88vm3OGibBwM4w4Q5HODivP9g+W3U+uzLinFKlaFRcnoub7zp/2mZEj+NVlraRs+s6DYzadNLbsVVriRVYBVwDt4YA9lHHAr5e/4KU/BvxB4h/ZI8FfEBZoW8M6Vqs9lJbo+97gSKFS7DjggSxMvOQRJnivoOfwxqnx/+OljHGuoxaG1qjXarL57ytJ8yrBJnKMFLAsMhfMPPTGz/wAF4Ndsvgp/wTm0jwzbxWtrNqFxb6bb2sa7ViQZzgew5/Cs51vY1adKk7yur/qj42nTeIjUxFbZJtPv5n4xzmQYkt1+2QyZUqDzg8HI+lcxretx6Fol3pMf2pbhpFNuxk4htznMTAjP90girei6hILGcbh8pLsoOOtR32rR3li0c0cc/mZYLIu7Z9D2/CvoKlNTVmeDTxM4ydzi0jj53bWNFdVceB7K+2yWt08KsozGwDbT9ciil7Nj9rHqRz2q3chU8bevvUOoWCxxOy8Egk+9FFakKTMe5h/0lV/56Rda+gP+Ceuoeb8Vr7QWXcNd09pEY/djkiG/JHoV3D8aKK48RFOlK57mQyax9Jruvx0Z+i/wI/4J66D+1T4Nvr4Xl1oepQ25X7XazGKV9pY7WIBDDk8kZxj0xWHcfseaL8CpNHk/tbxBrGoRXyu32q9/ccH0VV3fRgQaKK+ZqYmrFtKTPts0y/DOs5ciP0a/Z1+HGj/DnwLY6rHaxtOsAcbVB2gDgAmvx8/4L6ftRal8b/2kNH8JkS2ul+H4zMI2xiSSQlA3XsA3/fQ9KKKjK/fxkebXc+YzSTjQaj6fI+CZLn7PY3kqjl38sD+6BxVSz3eag3H5hlvfiiivs472PlI6p3NW0nyp7dO1FFFUSf/Z"/>
```

Comment:
Even though @subfuzion is handsome, I'm not sure we want his face on every single email we send on behalf of amp :)